modules/ROOT/pages/contributing.adoc

```diff
@@ -0,0 +1,34 @@
+[[contributing]]
+= Contributing to Fedora Silverblue
+
+Help us improve Fedora Silverblue by contributing to the project!
+
+[[developing]]
+== Developing Silverblue
+
+If you are a developer and want to contribute, submit your pull request to https://github.com/projectatomic/rpm-ostree[rpm-ostree.]
```

Comment:
It's difficult to select just one project for developers to contribute to.  There's ostree, rpm-ostree, flatpak, podman, plus all of the general Fedora projects that Silverblue consumes as well.

I'd suggest listing the primary projects that I've noted above, as well as an additional note about contributing to Fedora in general.

Reply:
That makes sense. I have updated the page. Please check and let me know what you think. Thanks.

Reply:
Excellent revision!